streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -119,18 +119,6 @@ public void commit() {
         commitNeeded = false;
     }
 
-    /**
```

Comment:
Why do we remove the standby task close?

Reply:
We remove the `suspend` and `closeSuspended` because we no longer suspend standby tasks at all as of the [PR 7321](https://github.com/apache/kafka/pull/7321) but forgot to clean up all the methods